src/download-cid.js

```diff
@@ -0,0 +1,144 @@
+const { join } = require('path')
+const fs = require('fs-extra')
+const i18n = require('i18next')
+const isIPFS = require('is-ipfs')
+const { clipboard, app, shell } = require('electron')
+const logger = require('./common/logger')
+const { IS_MAC } = require('./common/consts')
+const setupGlobalShortcut = require('./setup-global-shortcut')
+const { selectDirectory } = require('./dialogs')
+const dock = require('./dock')
+const showPrompt = require('./prompt')
+const { showDialog } = require('./dialogs')
+
+const CONFIG_KEY = 'downloadHashShortcut'
+
+const SHORTCUT = IS_MAC
+  ? 'Command+Control+H'
+  : 'CommandOrControl+Alt+D'
+
+async function saveFile (dir, file) {
+  const location = join(dir, file.path)
+  await fs.outputFile(location, file.content)
+}
+
+async function getCID () {
+  const text = clipboard.readText().trim()
+
+  const { button, input } = await showPrompt({
+    title: i18n.t('downloadCidContentDialog.title'),
+    message: i18n.t('downloadCidContentDialog.message'),
+    defaultValue: isIPFS.cid(text) ? text : '',
```

Comment:
Automatically filling the field if CID is in clipboard is a REALLY nice experience!

`downloadCidContentDialog.message` says this could be a CID or a content path, so perhaps change this to:
```suggestion
    defaultValue: (isIPFS.cid(text) || isIPFS.path(text)) ? text : '',
```